blog/2023-09-08-newsletter.mdx

```diff
@@ -0,0 +1,195 @@
+---
+title: "Conda Newsletter, Issue #2"
+slug: "2023-09-08-newsletter"
```

Comment:
@Pinak-Datta, before we merge, might be good to change the dates everywhere to be closer to the actual date of the newsletter release. 2023-09-19 or 2023-09-20 would be fine.

Reply:
Changed that.